src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationSource.cs

```diff
@@ -23,6 +24,11 @@ public AzureAppConfigurationSource(Action<AzureAppConfigurationOptions> optionsI
 
         public IConfigurationProvider Build(IConfigurationBuilder builder)
         {
+            if (Environment.GetEnvironmentVariable(ConditionalProviderConstants.DisableProviderEnvironmentVariable) == "true")
```

Comment:
This can throw

Reply:
I meant reading the environment variable could throw Security Exception. We should catch and ignore it in that case.